Makefile

```diff
@@ -54,11 +54,11 @@ endif
 .PHONY: build
 build: clean submodules
 	mvn versions:set -DnewVersion=$(version) -DgenerateBackupPoms=false
-	mvn package
+	mvn verify
 	cp ./target/$(artifact_name)-$(version).jar ./$(artifact_name).jar
 
 .PHONY: dist
-dist: clean coverage submodules package
+dist: clean coverage submodules verify
```

Comment:
should this be changed to verify as well?  Don't we want it to do the package target?

Reply:
Yes maven will have to execute `package` before  `verify` since the phases are done in this order:
compile => test => package => verify

Reply:
of course